tests/lib/rule-tester/rule-tester.js

```diff
@@ -6213,7 +6213,7 @@ describe("RuleTester", () => {
 			} catch (error) {
 				const normalizedStack = normalizeStack(error);
 				assert.include(normalizedStack, "at RuleTester.run.valid[0]");
-				assert.include(normalizedStack, "at RuleTester.run.valid");
+				assert.include(normalizedStack, "at RuleTester.run.valid\n");
```

Comment:
This change is to ensure this part appears as a separate line. Without the added `\n`, it would already found the substring in the `at RuleTester.run.valid[0]` line.